src/server/package/src/model_explorer/apis.py

```diff
@@ -15,7 +15,6 @@
 
 from typing import TypedDict, Union
 
-import torch
```

Comment:
Should we do the `try...except` and `if torch is None...` thing here as well?

Reply:
Indeed, it would be required for type checking. Done, thanks!